explorer/src/cli/clnodes.ts

```diff
@@ -6,7 +6,7 @@ import {
 } from '../utils/constants'
 
 const EXPLORER_BASE_URL =
-  process.env.EXPLORER_BASE_URL || 'https://explorer.chain.link'
+  process.env.EXPLORER_BASE_URL || 'http://localhost:8080'
```

Comment:
Running locally (or by the docker-compose) explorer is exposed on `http://localhost:3001`. Is your default is set for `ngrok` setup?

Reply:
I've set this for `yarn dev:server` the docker-compose setup should set EXPLORER_BASE_URL right?

Reply:
Add `clnode` is not typically run from within docker-compose. So, no, this will not work with the compose setup - is that the preferred port?

Maybe we could use the same port in both places?

Reply:
Im ok to keep the 8080 port. The only downside is that i would have to manually change it in docker. What is typical way of running `clnode`?

Reply:
It's 8080 now in both places.